composer.json

```diff
@@ -1,37 +1,28 @@
 {
     "name": "dhii/espresso-interface",
-    "description": "The interfaces for Espresso.",
+    "description": "The interfaces for Espresso: a PHP expression builder and evaluation library.",
     "type": "library",
     "license": "MIT",
     "authors": [
         {
-            "name": "Dhii",
+            "name": "Dhii Team",
             "email": "development@dhii.co"
         }
     ],
     "minimum-stability": "dev",
     "require": {
         "php": "^5.3.9 | ^7.0",
```

Comment:
I usually use `5.3.9` in interface packages regardless of whether there is method overriding or not.  Probably a bad habit of mine.

`ContextInterface#getValue()` seems to be a remnant of when `ContextInterface` didn't extend `ValueAwareInterface`. Looks like I forgot about it.